features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioPalSecurityManager.cpp

```diff
@@ -484,6 +515,11 @@ bool CordioSecurityManager::sm_handler(const wsfMsgHdr_t* msg) {
             connection_handle_t connection = evt->hdr.param;
 
             if (evt->oob) {
+            	// FIXME: Nothing in the API indicates if smp or sc OOB are
```

Comment:
This is where the fun is.

Reply:
the pairing must know what kind of pairing it is (if both sides support it it will always be SC)

Reply:
Do we need to discriminate OOB request at this level ? 
Would it make sense to send an event requesting OOB and upper layer responding with either legacy OOB or LESC OOB; depending on what has been exchanged ?